lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -219,6 +277,48 @@ def infer(self, image: Image.Image, threshold: float) -> list[Element]:
             elements.append(element)
         return elements
 
+    def batch_infer(self, images: List[Image.Image], threshold: float) -> List[List[Element]]:
```

Comment:
couldn't the batch and batch_infer merged?

Reply:
Ok

Reply:
Merged.